yarn.lock

```diff
@@ -4541,6 +4557,16 @@ react-timeago@^3.3.0:
   version "3.4.3"
   resolved "https://registry.yarnpkg.com/react-timeago/-/react-timeago-3.4.3.tgz#eb9061eefb044e4a2b09ce8c99d34645b2dbfa25"
 
+react-transition-group@1.x:
```

Comment:
update this since you change the package.json

either try `yarn` or if that doesn't work (it should tho?) just nuke the lockfile and run `yarn` again